src/js/stores/UserSettingsStore.ts

```diff
@@ -2,19 +2,23 @@ import EventEmitter from "events";
 
 import LocalStorageUtil from "../utils/LocalStorageUtil";
 
-const LOCAL_STORAGE_KEY = "dcosUserSettings";
+const LOCAL_STORAGE_KEY: string = "dcosUserSettings";
+const REFRESH_RATE_SETTING: string = "RefreshRate";
 
 function getLocalStorageObject() {
   const localStorageObject = LocalStorageUtil.get(LOCAL_STORAGE_KEY);
-  try {
-    return JSON.parse(localStorageObject);
-  } catch (e) {
-    return null;
+  if (localStorageObject != null) {
+    try {
+      return JSON.parse(localStorageObject);
+    } catch (e) {
+      return null;
+    }
   }
+  return null;
 }
 
 class UserSettingsStore extends EventEmitter {
-  getKey(key) {
+  getKey(key: string): unknown {
```

Comment:
Is the return type really `unknown`?

Reply:
Yes since it is a catchall function it can basically return anything.